saos-webapp/src/main/resources/message/common_pl.properties

```diff
@@ -1,8 +1,11 @@
 saos.fullname = System Analizy Orzeczeń Sądowych
 
-pageNotFound = Page not found
 choose = Wybierz
 
+error.pageNotFound = Page not found
+error.internal = Internal error occurred
```

Comment:
A good idea with these codes. However, I would change the polish ones to:
- Brak strony pod danym adresem
- Wystąpił błąd aplikacji